<?xml version="1.0" ?>
<testsuites disabled="0" errors="0" failures="65" tests="66" time="0.0">
	<testsuite disabled="0" errors="0" failures="23" name="flake8._\src\data_helper_py" skipped="0" tests="23" time="0" file=".\src\data_helper.py">
		<testcase name="E302, expected 2 blank lines, found 1" file=".\src\data_helper.py" line="5">
			<failure type="failure" message=".\src\data_helper.py:5:1: E302 expected 2 blank lines, found 1"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\data_helper.py" line="5">
			<failure type="failure" message=".\src\data_helper.py:5:31: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="Q002, Remove bad quotes from docstring" file=".\src\data_helper.py" line="6">
			<failure type="failure" message=".\src\data_helper.py:6:5: Q002 Remove bad quotes from docstring"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\data_helper.py" line="10">
			<failure type="failure" message=".\src\data_helper.py:10:8: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="E302, expected 2 blank lines, found 0" file=".\src\data_helper.py" line="12">
			<failure type="failure" message=".\src\data_helper.py:12:1: E302 expected 2 blank lines, found 0"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\data_helper.py" line="12">
			<failure type="failure" message=".\src\data_helper.py:12:83: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="Q002, Remove bad quotes from docstring" file=".\src\data_helper.py" line="13">
			<failure type="failure" message=".\src\data_helper.py:13:5: Q002 Remove bad quotes from docstring"/>
		</testcase>
		<testcase name="E231, missing whitespace after ','" file=".\src\data_helper.py" line="28">
			<failure type="failure" message=".\src\data_helper.py:28:19: E231 missing whitespace after ','"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\data_helper.py" line="29">
			<failure type="failure" message=".\src\data_helper.py:29:21: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\data_helper.py" line="30">
			<failure type="failure" message=".\src\data_helper.py:30:25: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="Q000, Remove bad quotes" file=".\src\data_helper.py" line="32">
			<failure type="failure" message=".\src\data_helper.py:32:27: Q000 Remove bad quotes"/>
		</testcase>
		<testcase name="Q000, Remove bad quotes" file=".\src\data_helper.py" line="32">
			<failure type="failure" message=".\src\data_helper.py:32:46: Q000 Remove bad quotes"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\data_helper.py" line="35">
			<failure type="failure" message=".\src\data_helper.py:35:25: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\data_helper.py" line="37">
			<failure type="failure" message=".\src\data_helper.py:37:28: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\data_helper.py" line="41">
			<failure type="failure" message=".\src\data_helper.py:41:28: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="E226, missing whitespace around arithmetic operator" file=".\src\data_helper.py" line="41">
			<failure type="failure" message=".\src\data_helper.py:41:50: E226 missing whitespace around arithmetic operator"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\data_helper.py" line="43">
			<failure type="failure" message=".\src\data_helper.py:43:25: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\data_helper.py" line="44">
			<failure type="failure" message=".\src\data_helper.py:44:23: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\data_helper.py" line="49">
			<failure type="failure" message=".\src\data_helper.py:49:48: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="Q002, Remove bad quotes from docstring" file=".\src\data_helper.py" line="50">
			<failure type="failure" message=".\src\data_helper.py:50:5: Q002 Remove bad quotes from docstring"/>
		</testcase>
		<testcase name="E203, whitespace before ','" file=".\src\data_helper.py" line="54">
			<failure type="failure" message=".\src\data_helper.py:54:61: E203 whitespace before ','"/>
		</testcase>
		<testcase name="Q000, Remove bad quotes" file=".\src\data_helper.py" line="54">
			<failure type="failure" message=".\src\data_helper.py:54:64: Q000 Remove bad quotes"/>
		</testcase>
		<testcase name="W391, blank line at end of file" file=".\src\data_helper.py" line="58">
			<failure type="failure" message=".\src\data_helper.py:58:1: W391 blank line at end of file"/>
		</testcase>
	</testsuite>
	<testsuite disabled="0" errors="0" failures="29" name="flake8._\src\history_helper_py" skipped="0" tests="29" time="0" file=".\src\history_helper.py">
		<testcase name="E302, expected 2 blank lines, found 1" file=".\src\history_helper.py" line="5">
			<failure type="failure" message=".\src\history_helper.py:5:1: E302 expected 2 blank lines, found 1"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\history_helper.py" line="8">
			<failure type="failure" message=".\src\history_helper.py:8:17: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\history_helper.py" line="9">
			<failure type="failure" message=".\src\history_helper.py:9:17: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="E111, indentation is not a multiple of four" file=".\src\history_helper.py" line="22">
			<failure type="failure" message=".\src\history_helper.py:22:8: E111 indentation is not a multiple of four"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\history_helper.py" line="22">
			<failure type="failure" message=".\src\history_helper.py:22:16: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="E111, indentation is not a multiple of four" file=".\src\history_helper.py" line="34">
			<failure type="failure" message=".\src\history_helper.py:34:10: E111 indentation is not a multiple of four"/>
		</testcase>
		<testcase name="E117, over-indented" file=".\src\history_helper.py" line="34">
			<failure type="failure" message=".\src\history_helper.py:34:10: E117 over-indented"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\history_helper.py" line="34">
			<failure type="failure" message=".\src\history_helper.py:34:26: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="Q000, Remove bad quotes" file=".\src\history_helper.py" line="34">
			<failure type="failure" message=".\src\history_helper.py:34:32: Q000 Remove bad quotes"/>
		</testcase>
		<testcase name="E111, indentation is not a multiple of four" file=".\src\history_helper.py" line="35">
			<failure type="failure" message=".\src\history_helper.py:35:10: E111 indentation is not a multiple of four"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\history_helper.py" line="35">
			<failure type="failure" message=".\src\history_helper.py:35:26: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="Q000, Remove bad quotes" file=".\src\history_helper.py" line="35">
			<failure type="failure" message=".\src\history_helper.py:35:33: Q000 Remove bad quotes"/>
		</testcase>
		<testcase name="E303, too many blank lines (2)" file=".\src\history_helper.py" line="38">
			<failure type="failure" message=".\src\history_helper.py:38:5: E303 too many blank lines (2)"/>
		</testcase>
		<testcase name="E231, missing whitespace after ','" file=".\src\history_helper.py" line="40">
			<failure type="failure" message=".\src\history_helper.py:40:45: E231 missing whitespace after ','"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\history_helper.py" line="43">
			<failure type="failure" message=".\src\history_helper.py:43:28: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="Q002, Remove bad quotes from docstring" file=".\src\history_helper.py" line="44">
			<failure type="failure" message=".\src\history_helper.py:44:5: Q002 Remove bad quotes from docstring"/>
		</testcase>
		<testcase name="E302, expected 2 blank lines, found 1" file=".\src\history_helper.py" line="51">
			<failure type="failure" message=".\src\history_helper.py:51:1: E302 expected 2 blank lines, found 1"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\history_helper.py" line="51">
			<failure type="failure" message=".\src\history_helper.py:51:31: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="Q002, Remove bad quotes from docstring" file=".\src\history_helper.py" line="52">
			<failure type="failure" message=".\src\history_helper.py:52:5: Q002 Remove bad quotes from docstring"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\history_helper.py" line="57">
			<failure type="failure" message=".\src\history_helper.py:57:13: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\history_helper.py" line="58">
			<failure type="failure" message=".\src\history_helper.py:58:6: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="E231, missing whitespace after ','" file=".\src\history_helper.py" line="59">
			<failure type="failure" message=".\src\history_helper.py:59:10: E231 missing whitespace after ','"/>
		</testcase>
		<testcase name="E111, indentation is not a multiple of four" file=".\src\history_helper.py" line="60">
			<failure type="failure" message=".\src\history_helper.py:60:8: E111 indentation is not a multiple of four"/>
		</testcase>
		<testcase name="E302, expected 2 blank lines, found 1" file=".\src\history_helper.py" line="63">
			<failure type="failure" message=".\src\history_helper.py:63:1: E302 expected 2 blank lines, found 1"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\history_helper.py" line="63">
			<failure type="failure" message=".\src\history_helper.py:63:36: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="Q002, Remove bad quotes from docstring" file=".\src\history_helper.py" line="64">
			<failure type="failure" message=".\src\history_helper.py:64:5: Q002 Remove bad quotes from docstring"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\history_helper.py" line="68">
			<failure type="failure" message=".\src\history_helper.py:68:8: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\history_helper.py" line="69">
			<failure type="failure" message=".\src\history_helper.py:69:10: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="W391, blank line at end of file" file=".\src\history_helper.py" line="72">
			<failure type="failure" message=".\src\history_helper.py:72:1: W391 blank line at end of file"/>
		</testcase>
	</testsuite>
	<testsuite disabled="0" errors="0" failures="10" name="flake8._\src\image_helper_py" skipped="0" tests="10" time="0" file=".\src\image_helper.py">
		<testcase name="F401, 'src.history_helper.history' imported but unused" file=".\src\image_helper.py" line="1">
			<failure type="failure" message=".\src\image_helper.py:1:1: F401 'src.history_helper.history' imported but unused"/>
		</testcase>
		<testcase name="F811, redefinition of unused 'history' from line 1" file=".\src\image_helper.py" line="5">
			<failure type="failure" message=".\src\image_helper.py:5:15: F811 redefinition of unused 'history' from line 1"/>
		</testcase>
		<testcase name="E231, missing whitespace after ','" file=".\src\image_helper.py" line="5">
			<failure type="failure" message=".\src\image_helper.py:5:22: E231 missing whitespace after ','"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\image_helper.py" line="5">
			<failure type="failure" message=".\src\image_helper.py:5:31: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="Q002, Remove bad quotes from docstring" file=".\src\image_helper.py" line="6">
			<failure type="failure" message=".\src\image_helper.py:6:5: Q002 Remove bad quotes from docstring"/>
		</testcase>
		<testcase name="Q000, Remove bad quotes" file=".\src\image_helper.py" line="12">
			<failure type="failure" message=".\src\image_helper.py:12:29: Q000 Remove bad quotes"/>
		</testcase>
		<testcase name="Q000, Remove bad quotes" file=".\src\image_helper.py" line="13">
			<failure type="failure" message=".\src\image_helper.py:13:24: Q000 Remove bad quotes"/>
		</testcase>
		<testcase name="E225, missing whitespace around operator" file=".\src\image_helper.py" line="14">
			<failure type="failure" message=".\src\image_helper.py:14:17: E225 missing whitespace around operator"/>
		</testcase>
		<testcase name="E231, missing whitespace after ','" file=".\src\image_helper.py" line="14">
			<failure type="failure" message=".\src\image_helper.py:14:39: E231 missing whitespace after ','"/>
		</testcase>
		<testcase name="Q000, Remove bad quotes" file=".\src\image_helper.py" line="14">
			<failure type="failure" message=".\src\image_helper.py:14:40: Q000 Remove bad quotes"/>
		</testcase>
	</testsuite>
	<testsuite disabled="0" errors="0" failures="0" name="flake8._\test\test_data_helper_py" skipped="0" tests="1" time="0" file=".\test\test_data_helper.py">
		<testcase name="Check passed" file=".\test\test_data_helper.py"/>
	</testsuite>
	<testsuite disabled="0" errors="0" failures="2" name="flake8._\test\test_history_helper_py" skipped="0" tests="2" time="0" file=".\test\test_history_helper.py">
		<testcase name="E201, whitespace after '('" file=".\test\test_history_helper.py" line="57">
			<failure type="failure" message=".\test\test_history_helper.py:57:46: E201 whitespace after '('"/>
		</testcase>
		<testcase name="E231, missing whitespace after ','" file=".\test\test_history_helper.py" line="57">
			<failure type="failure" message=".\test\test_history_helper.py:57:66: E231 missing whitespace after ','"/>
		</testcase>
	</testsuite>
	<testsuite disabled="0" errors="0" failures="1" name="flake8._\test\test_image_helper_py" skipped="0" tests="1" time="0" file=".\test\test_image_helper.py">
		<testcase name="W391, blank line at end of file" file=".\test\test_image_helper.py" line="110">
			<failure type="failure" message=".\test\test_image_helper.py:110:1: W391 blank line at end of file"/>
		</testcase>
	</testsuite>
</testsuites>

